community/self-hosting-novu/deploy-with-docker.mdx

```diff
@@ -158,3 +158,29 @@ This would producte the following url: - API: company.com/novu-api
   These env variables should be present on all services novu provides due to tight
   coupling.
 </Note>
+
+## FAQs
+### Local Tunnel and Self-Hosted Deployments
+
+Local tunnel support is not included as part of the Novu Docker images. This is typically unnecessary for many users, depending on how their environment is structured.
```

Comment:
Please prepend the following first sentence: "Novu uses a local tunnel to connect to Local Studio during development."